tools/report-converter/codechecker_report_converter/cpplint/analyzer_result.py

```diff
@@ -0,0 +1,36 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+
+from codechecker_report_converter.analyzer_result import AnalyzerResult
+
+from .output_parser import CpplintParser
+from ..plist_converter import PlistConverter
+
+
+class CpplintAnalyzerResult(AnalyzerResult):
+    """ Transform analyzer result of cpplint. """
+
+    TOOL_NAME = 'cpplint'
+    NAME = 'cpplint'
+    URL = 'https://github.com/cpplint/cpplint'
```

Comment:
Please update the user guide with this new tool:
https://github.com/Ericsson/codechecker/blob/master/tools/report-converter/README.md

Also update the main readme file: https://github.com/Ericsson/codechecker/blob/master/docs/README.md#storage-of-reports-from-analyzer-tools